app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,38 @@
+@import "materialize/components/color";
+$primary-color: color("blue", "lighten-2") !default;
+@import "materialize";
+
+$mainText: #e8eaf6; 
+
+.wrapper {
+  min-height: 70vh;
+}
+
+footer {
+  height:100%;
+  width: 100%;
+  margin-top: 0px;
+}
+
+.page-footer {
+  margin-top: 0px !important;
+}
+
+.sidebar {
+  padding-bottom: 0px;
```

Comment:
Properties should be ordered min-height, padding-bottom<br>`0px` should be written without units as `0`